### PR TITLE
Initial change

### PR DESCRIPTION
**Pull request by jbelgard**

### CODE REVIEW

File: friends/src/App.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import './App.css';
+import Data from './data';
+import axios from 'axios';
+import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
```
Comment (JaySebagh) [inline friends/src/App.js]: What is the Router stuff for? I don't see any link or such.

File: README.md [hunk 2]
```diff
@@ -4,8 +4,8 @@ Topics:
```
Comment (sebaghdev) [inline README.md:4]: Don't forget to check your errors.

> Compiled with warnings.
> 
> ./src/components/AddFriendForm.js
>   Line 1:  'Component' is defined but never used  no-unused-vars
> 
> ./src/data.js
>   Line 1:  'Component' is defined but never used  no-unused-vars
>   Line 2:   'axios' is defined but never used      no-unused-vars

File: friends/src/App.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        console.log(err);
+      });
+  };
+  render() {
+    return (
+      <Router>
+        <div className='App'>
+          <Data friends={this.state.friends}/>
+          <Link to='/new-friend'><button>add Atlantian</button></Link>
+          <Route exact path='/new-friend' render={props => <AddFriendForm {...props} addFriend={this.addFriend} />}/>
```
Comment (sebaghdev) [inline friends/src/App.js]: Remember to break it down into multiple lines for readability.

File: friends/src/data.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
```
Comment (sebaghdev) [inline friends/src/data.js]: I would not name this file data. That makes things confusing. Data file usually holds data like an array of object. Also would want to have this inside of your components.

File: friends/src/components/AddFriendForm.js [hunk 5]
```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react'
+
+const Hidden = {
+    display: 'none'
+}
+
+class AddFriendForm extends React.Component {
```
Comment (sebaghdev) [inline friends/src/components/AddFriendForm.js]: No need to write out React.Components if you de-structured it on line 1.

File: friends/src/components/AddFriendForm.js [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+        this.setState({
+            friend: {
+                name: '',
+                age: '',
+                email: '',
+            }
+        });
+    };
+    render() {
+        console.log(this.state.friend)
```
Comment (sebaghdev) [inline friends/src/components/AddFriendForm.js]: Don't forget to remove console.logs when you're done with them.

**Pull request merged**